ocaml/idl/datamodel.ml

```diff
@@ -6659,6 +6663,16 @@ let pool_create_VLAN = call
     ~allowed_roles:_R_POOL_OP
     ()
 
+let pool_management_reconfigure = call
+    ~name:"management_reconfigure"
+    ~in_oss_since:None
+    ~in_product_since:rel_falcon
+    ~params:[
+             Ref _network, "network", "network to which all hosts management to be reside in the Pool";
```

Comment:
Given that we also have the `~doc` string, I think we can just leave this as "The network" and the meaning will be obvious from context.